Code/Framework/AzCore/AzCore/Component/ComponentApplicationBus.h

```diff
@@ -52,6 +53,7 @@ namespace AZ
             Tool = 1 << 1,
             Game = 1 << 2,
             Headless = 1 << 3,
+            ConsoleMode = 1 << 4,
```

Comment:
Can you add an explanation per mode as to which use cases will use them.

Reply:
Sure, i'll also have to reto-actively explain the existing modes as well